### ISSUE TITLE
Incompatible with Node between 9.0.0 and <= 9.10.0?

### ISSUE THREAD

**Issue by OliverJAsh** [eslint/typescript-eslint-parser#482]
``` sh-session
$ yarn add -D typescript-eslint-parser
yarn add v1.6.0
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error typescript-eslint-parser@16.0.0: The engine "node" is incompatible with this module. Expected version "^6.14.0 || ^8.10.0 || >=9.10.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.

$ node --version
v9.8.0
```

Is this intentional?

**Comment by JamesHenry** [comment]
Tbh I just copied eslint/eslint - I am not sure of the reason behind it.

It was probably a mistake to do that, if you don’t mind submitting a PR to loosen it again, I’ll get it merged.

**Comment by platinumazure** [comment]
We did it on the ESLint side to try to allow as many features as possible in the codebase-- sometimes a new feature on 10.x might get backported to a late version in 9.x. So we tried to minimize the chance of compatibility issues by using later version of the non-LTS branches.

**Comment by OliverJAsh** [comment]
@platinumazure Interesting, thanks.

@JamesHenry Do you want to stick with that, or can I raise a PR to lower the range? I'm using 9.8.0…

**Comment by JamesHenry** [comment]
It's a tricky one - Is there a specific reason why you are using 9.8 and not 9.10? I imagine not, you just aren't.

So you could easily upgrade node, but on the other hand it's just another point of friction to using the lib, so avoiding it might be best...

**Comment by JamesHenry** [comment]
I'm going to loosen it again

**Comment by OliverJAsh** [comment]
Would it be possible to release this?

**Comment by Jungwoo-An** [comment]
occurred error as same issue.

```bash
typescript-eslint-parser@16.0.0: The engine "node" is incompatible with this module. Expected version "^6.14.0 || ^8.10.0 || >=9.10.0".
```

my node version: v8.9.4

**Comment by JamesHenry** [comment]
Sorry for the delay, @OliverJAsh @Jungwoo-An, 16.0.1 is out now

**Comment by JoaquinAraujo** [comment]
![image](https://user-images.githubusercontent.com/20958348/44897601-3a858400-acca-11e8-9af0-e524f94f0efa.png)

**Comment by platinumazure** [comment]
@JoaquinAraujo The error you're seeing is for the eslint package, not typescript-eslint-parser. ESLint has not loosened their compatibility, but typescript-eslint-parser has.
